test/integration/connection_lost_test.exs

```diff
@@ -27,4 +27,37 @@ defmodule MongoosePush.API.ConnectionTest do
       Process.sleep(1000)
     end)
   end
+
+  test "Metrics can be correctly received from Prometheus" do
+    # Push some notification to trigger metrics
+    %{status_code: status_code, body: body} =
+      HTTPoison.post!(
```

Comment:
I think we already have helpers that do such requests in `MongoosePush.Support.API` module.